src/main/kotlin/com/hypto/iam/server/db/repositories/ActionRepo.kt

```diff
@@ -42,8 +41,7 @@ object ActionRepo : BaseRepo<ActionsRecord, Actions, String>() {
 
     suspend fun update(hrn: ActionHrn, description: String): ActionsRecord? {
         val condition = ACTIONS.HRN.eq(hrn.toString())
-        val dao = dao()
-        return dao.ctx().update(dao.table)
+        return ctx("action.update").update(ACTIONS)
```

Comment:
nit: we can get the method name from getStackTrace as well so that if we change the method in future, it gets updated automatically.